src/Components/AddTask/AddTask.tsx

```diff
@@ -0,0 +1,175 @@
+// nadia.mohamed.taha166@gmail.com
+// @Password123!
+import axios from "axios";
+import { useContext, useEffect, useState } from "react";
+import { useNavigate } from "react-router-dom";
+import { AuthContext } from "./../../Context/AuthContext";
+import { ToastContext } from "../../Context/ToastContext";
+import { SubmitHandler, useForm } from "react-hook-form";
+import { ProjectContext } from '../../Context/ProjectContext';
+import { FormValues, projectType } from './../../Types/Types';
+
+
+const AddTask: React.FC = () => {
+  const { baseUrl, requestHeaders }: any = useContext(AuthContext);
+  const { getToastValue }: any = useContext(ToastContext);
+  const {  projects }: any = useContext(ProjectContext);
+  // console.log(projects, "from addtasks");
+  const [isLoading, setIsLoading] = useState(false);
```

Comment:
you can add types to the use state like that:
useState<boolean>(false);

this will prevent developers from setting unexpected values such as setIsLoading("loading")

Reply:
I meant here useState<boolean>(false), but it seems that there is a bug in GitHub editing the text.